src/vinylcontrol/vinylcontrolxwax.h

```diff
@@ -135,6 +138,8 @@ class VinylControlXwax : public VinylControl {
     // The drift between the vinyl position and the file position from the most
     // recent run of analyzeSamples.
     double m_dDriftAmt;
+    double m_relativedDriftAmtMem;
```

Comment:
`Mem` is not a common postfix. What does it mean?

Reply:
@Swiftb0y In my head, `Mem` means `Memory`. It is to keep track of the drift amount at some point and only update it with some specific condition (needle drop, passthrough mode etc.). I didn't find a better name.

Reply:
What about `m_initialRelativeDriftAmt`?

Reply:
sure...